users/src/commonMain/kotlin/majestic/users/profile/roles/campus/CampusIcon.kt

```diff
@@ -0,0 +1,35 @@
+package majestic.users.profile.roles.campus
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Icon
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.unit.dp
+import majestic.users.profile.roles.CampusColors
+import org.jetbrains.compose.resources.painterResource
+import tz.co.asoft.majestic_users.generated.resources.Res
+import tz.co.asoft.majestic_users.generated.resources.ic_school
+
+@Composable
+fun CampusIcon(colors: CampusColors) {
```

Comment:
Move the modifier definition to the call site and consider trying to achieve this by not wrapping it inside another Box composable!